server/app/views/v1/host_nodes/_host_node.json.jbuilder

```diff
@@ -1,4 +1,5 @@
 json.id node.node_id
+json.token node.token
```

Comment:
Moved the token out to a separate `/v1/nodes/:grid/:node/token` API, and limited both `POST /v1/grids/:grid/nodes` and  `GET /v1/nodes/:grid/:name/token` to grid admins.